backend/audits/tests/test_tasks.py

```diff
@@ -0,0 +1,71 @@
+# from django_celery_beat.models import PeriodicTask
+# from django.core.management import call_command
+import httpretty
+import json
+from django.test import TestCase, override_settings
+from audits.tasks import request_audit
+from projects.models import (
+    Page,
+    Project,
+    AvailableAuditParameters,
+    ProjectAuditParameters,
+)
+from audits.models import Audit, AuditStatusHistory
+from unittest.mock import patch
+from .mocks import poll_audit_results
+
+
+class TasksTestCase(TestCase):
+    @httpretty.activate
+    @override_settings(CELERY_EAGER=True)
+    @patch("audits.tasks.poll_audit_results", poll_audit_results)
+    def test_request_audit(self):
+        httpretty.register_uri(
+            httpretty.POST,
+            "https://www.webpagetest.org/runtest.php",
+            body='{ "statusCode": 200, "statusText": "Ok", "data": { "testId": "191024_HA_976b046886025ec8693cbe4f1145929e", "ownerKey": "9259cab7a56de88af4f67e16e631ed954d8c9fee", "jsonUrl": "https://www.webpagetest.org/jsonResult.php?test=191024_HA_976b046886025ec8693cbe4f1145929e", "xmlUrl": "https://www.webpagetest.org/xmlResult/191024_HA_976b046886025ec8693cbe4f1145929e/", "userUrl": "https://www.webpagetest.org/result/191024_HA_976b046886025ec8693cbe4f1145929e/", "summaryCSV": "https://www.webpagetest.org/result/191024_HA_976b046886025ec8693cbe4f1145929e/page_data.csv", "detailCSV": "https://www.webpagetest.org/result/191024_HA_976b046886025ec8693cbe4f1145929e/requests.csv" } }',
```

Comment:
Can that body be set inside a JSON file for clarity? We could have e.g. `audits/tests/json_mocks/wpt_runtest_post_body.json` and `audits/tests/json_mocks/wpt_runtest_post_result.json`